tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,61 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+from robottelo.utils.datafactory import gen_string
+
+
+def test_positive_verify_default_values_for_global_registration(
+    session,
+    module_target_sat,
+    default_org,
+    default_location,
+    default_os,
+):
+    """Check for all the Default values pre-populated in the global registration template
+
+    :id: 34122bf3-ae23-47ca-ba3d-da0653d8fd33
+
+    :expectedresults: Some of the fields in the form should be populated
+        e.g. organization, location,remote-execution, insights setup.
+
+    :CaseLevel: Component
+
+    :steps:
+        1.Check for the default values in the global registration template
+    """
+    ak = module_target_sat.cli_factory.make_activation_key(
+        {'organization-id': default_org.id, 'name': gen_string('alpha')}
+    )
+    with module_target_sat.ui_session() as session:
+        session.organization.select(org_name=default_org.name)
+        session.location.select(loc_name=default_location.name)
```

Comment:
default_org and default_loc should be autoselected for admin user
```suggestion

```

Reply:
I would still prefer to keep it.

Reply:
why? it'd take additional step and time to select org and loc here which isn't even required

Reply:
Just being double sure, updated anyway.